com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/ShaderPass/ShaderPassPathTracing.hlsl

```diff
@@ -75,7 +75,10 @@ void ComputeSurfaceScattering(inout PathIntersection pathIntersection : SV_RayPa
 
 #ifndef SHADER_UNLIT
 
-    // Let's compute the world space position (the non-camera relative one if camera relative rendering is enabled)
+    // Override the geometric normal (otherwise, it is merely the non-mapped smooth normal)
+    GetCurrentIntersectionGeometricNormal(attributeData, bsdfData.geomNormalWS);
```

Comment:
hmm... there is a problem with this approach. You don't consider anymore the normal provided by the DCC tools or the normal computed by Morten Mikkelsen algorithm's.  What is happening in this case? 
Have you check what is happening with UV mirroring?

Reply:
This code snippet is only used in path tracing, in which context this geomNormal is used exclusively for geometric tests (is the ray shot above or below the triangle's plane). All shading computations are still performed with the shading normal.
This being said, the orientation needed to be checked against the potentially flipped input normal (when dbl-sided); this is now done.